src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs

```diff
@@ -810,9 +812,11 @@ await using (var stream = await GetInputStreamAsync(decompressionStream, snapsho
                     databaseRecord.QueueEtls = smugglerDatabaseRecord.QueueEtls;
                     databaseRecord.QueueConnectionStrings = smugglerDatabaseRecord.QueueConnectionStrings;
                     databaseRecord.IndexesHistory = smugglerDatabaseRecord.IndexesHistory;
+                    databaseRecord.PutSupportedFeaturesFromSmuggler(smugglerDatabaseRecord.SupportedFeatures);
 
                     // need to enable revisions before import
                     database.DocumentsStorage.RevisionsStorage.InitializeFromDatabaseRecord(smugglerDatabaseRecord);
+                    database.SupportedFeatures = new SupportedFeature(databaseRecord);
```

Comment:
we first initialize the database (at this point, the records from Smuggler have not yet been imported). then, we perform this action - and because the supported features may change, we need to apply those changes